src/docs/sdk/event-payloads/threads.mdx

```diff
@@ -26,13 +26,19 @@ attribute to cross-reference this thread.
 
 `crashed`
 
-: _Optional_. A flag indicating whether the thread crashed. Defaults to `false`.
+: _Optional_. A flag indicating whether the thread was the cause for the event being sent, e.g. due to a crash.
+Defaults to `false`.
 
 `current`
 
-: _Optional_. A flag indicating whether the thread was in the foreground.
+: _Optional_. A flag indicating whether the thread was in the foreground. A thread is in foreground when it's executing.
 Defaults to `false`.
 
+`ui`
```

Comment:
Before merging this, should we call it `main`?
Reason is here https://github.com/getsentry/rfcs/pull/36#discussion_r1024461690

Reply:
If I'm not mistaken, you could have multiple UI threads if your application supports multiple windows or screens. For that reason, I think keeping it as UI thread makes sense.

Reply:
I agree with @marandaneto, we should rename this to main thread. Android, iOS, .NET MAUI, etc. call this main thread. We could also have multiple main threads for applications with multiple windows.